src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/Command/VerifyCustomerAccount.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources xmlns="https://api-platform.com/schema/metadata/resources-3.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata/resources-3.0 https://api-platform.com/schema/metadata/resources-3.0.xsd"
+>
+    <resource
+        class="Sylius\Bundle\ApiBundle\Command\Account\VerifyCustomerAccount"
+        shortName="VerifyCustomerAccount"
+        routePrefix="shop"
+        messenger="input"
+        output="false"
+        status="202"
+    >
+        <validationContext>
+            <values>
+                <value name="groups">
+                    <values>
+                        <value>sylius</value>
+                    </values>
+                </value>
+            </values>
+        </validationContext>
+
+        <operations>
+            <operation
+                name="shop_resend_verification_email"
```

Comment:
Naming wise, this and command class should be changed to derivative of "Request verification token".

https://github.com/Sylius/Sylius/blob/a88c39a9de5a3650fc814de51f21bad37a0224d9/src/Sylius/Bundle/UserBundle/Controller/UserController.php#L160

```suggestion
                name="shop_request_verification"
```